tests/acceptance/usage-in-acceptance-test.js

```diff
@@ -0,0 +1,33 @@
+import { test } from 'qunit';
+import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+import { visit, click, fillIn, waitUntil } from 'ember-native-dom-helpers/test-support/helpers';
+
+moduleForAcceptance('Acceptance | usage in acceptance');
+
+test('Usage awaiting the world to settle', async function(assert) {
```

Comment:
No, babel 6 will know what to transpile automatically based on the browsers you support.